x/token/event.go

```diff
@@ -1,28 +1,13 @@
 package token
 
 import (
-	"fmt"
-	"sort"
 	"strings"
-
-	sdk "github.com/line/lbm-sdk/types"
 )
 
 const (
-	prefixEventType    = "EVENT_TYPE_"
```

Comment:
Is there any side-effect that might happen soon when someone uses the same prefix in the future again?
It's very unlikely It's okay to remove it.

Reply:
The prefix was there just for the legacy event's stringer. We don't support the legacy events anymore, I think it would be OK to remove it. In this PR, the relevant enums would also be removed.